front-end/assets/translations/pt-pt.csv

```diff
@@ -0,0 +1,195 @@
+Key;Translation
+ok;Ok
+select;Selecionar
+enabled;activar
+disabled;desativar
+cancel;cancelar
+done;Feito
+loading;Carregar
+documentation;Documenta��o

```

Comment:
Seems like there is some encoding problem in your file

Reply:
Ups my mistake

new version